.github/ISSUE_TEMPLATE/dependency-update-template.md

```diff
@@ -25,10 +25,11 @@ See outstanding dependabot[PRs](https://github.com/department-of-veterans-affair
 ## Engineering Checklist
 
 - [ ] Review these [PRs](https://github.com/department-of-veterans-affairs/notification-utils/pulls) to see which files need to change (do not modify those PRs; do not assume any specific file)
-- [ ] [setup.py](https://github.com/department-of-veterans-affairs/notification-utils/blob/main/setup.py) seems to be out of scope for dependabot, review it and ensure that the versions there match the versions in the requirements_for_test file 
-- [ ] Modify the files to update to the latest versions
-- [ ] Unit tests pass
-- [ ] Modify the Notification API pyproject.toml file to pull in this branch/sha and then deploy that API branch
+- [ ] [setup.py](https://github.com/department-of-veterans-affairs/notification-utils/blob/main/setup.py) seems to be out of scope for dependabot. Review all dependencies in `setup.py` and update them to latest if possible. Ensure that any dependencies that exists both in the `setup.py` and `requirements_for_test.py` file are the same version.
+- [ ] Increment Notification-Utils version number in `version.py`
+- [ ] Confirm all Notification-Utils unit tests pass locally
+- [ ] Modify the Notification API `pyproject.toml` file to pull in this branch/sha and then deploy that API branch. Confirm the regression tests pass. 
+- [ ] Review the Notification-Utils [README.md](https://github.com/department-of-veterans-affairs/notification-utils?tab=readme-ov-file#versioning) for expected versioning and tagging process. 
```

Comment:
The readme could be improved so that it clear a) how to cut and push the tag and b) how to create the release notes. If you would rather just put the steps in this template here, I'm good with that too.